packages/core/useColorMode/index.test.ts

```diff
@@ -16,7 +17,7 @@ describe('useColorMode', () => {
   })
 
   beforeEach(() => {
-    usePreferredDark().value = false
+    (usePreferredDark() as Ref<boolean>).value = false
```

Comment:
I am not sure why we tested like this before. But isn't that the change to `computed` make this statement invalid?

Reply:
This is for type-checking.

The value is mocked as a ref. https://github.com/vueuse/vueuse/pull/4403/files#diff-7811d4dbacf020831d35d217e1b1d3036a472d6a79ede4e1172d06cf9b58937dR12-R16

Reply:
<img width="987" alt="image" src="https://github.com/user-attachments/assets/19334a58-215c-489a-98d7-6bccc2a88598" />

Reply:
I’m not an expert in mocking Vitest 🤔 Could you share a pseudocode example?